modules/microshift-install-rpms.adoc

```diff
@@ -31,13 +31,6 @@ $ sudo subscription-manager repos \
 $ sudo dnf install -y microshift
 ----
 
-. Optional: Install greenboot for {product-title} by running the following command:
```

Comment:
Hi @ShaunaDiaz . This command was removed because Greenboot is now automatically installed on OCP?

Reply:
@dfitzmau Yes it is required, and no, this is for MicroShift. It has nothing to do with OCP.

Reply:
OK. So Greenboot is now automatically installed on MicroShift?

Reply:
Command removal is OK as now Greenboot is installed on MicroShift.